infrastructure/cdn-in-a-box/Makefile

```diff
@@ -164,6 +180,9 @@ $(ORT_RPM_ABSOLUTE): $(ORT_DIST_RPM)
 	cp -f "$?" "$@"
 
 # Dist rpms
+$(ATS_DIST_RPM): $(ATS_SOURCE)
+	docker-compose -f ../../cache-config/testing/docker/docker-compose-ats-build.yml build --parallel trafficserver_build && docker-compose -f ../../cache-config/testing/docker/docker-compose-ats-build.yml run --rm trafficserver_build
```

Comment:
Don't these `../../`s need to be expanded from some variable?

Reply:
Expanding from some variable in fdb6f38212.